controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1503,7 +1503,7 @@ int cli_bml::analyzer_init(std::string remote_pc_ip)
                 break;
             }
         }
-        if (t < 19) {
+        if (t < 1) {
```

Comment:
It's not needed anymore with the "controller: remove "unknown IP" states" commit (currently d0b9cea).
It was only needed because the controller didn't send updates to the analyzer when an event occurred (a client connected, disconnected,...).

Reply:
@morantr please remove this commit in that case

Reply:
done